zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -276,19 +276,7 @@ public void setReturn(InterpreterResult value, Throwable t) {
 
   @Override
   public Object clone() throws CloneNotSupportedException {
-    Paragraph paraClone = (Paragraph) super.clone();
-    Map<String, Object> config = new HashMap<>(this.getConfig());
-    // Show the editor by default
-    String hideEditorKey = "editorHide";
```

Comment:
ok that makes sense. could you include a brief line on this change in ZEPPELIN-420?